sklearn/decomposition/_lda.py

```diff
@@ -320,6 +322,25 @@ class conditional densities to the data and using Bayes' rule.
            [0.15297572, 0.00362644, 0.44412786, 0.39568399, 0.003586  ]])
     """
 
+    _parameter_constraints = {
+        "n_components": [Interval(Integral, 0, None, closed="neither")],
+        "doc_topic_prior": [None, Interval(Real, 0, 1, closed="both")],
+        "topic_word_prior": [None, Interval(Real, 0, 1, closed="both")],
+        "learning_method": [StrOptions({"batch", "online"})],
+        "learning_decay": [Interval(Real, 0, 1, closed="both")],
+        "learning_offset": [Interval(Real, 0, None, closed="left")],
```

Comment:
It seems that the docstring states that it should be greater than 1.0.

```suggestion
        "learning_offset": [Interval(Real, 1.0, None, closed="left")],
```

Having a number between 0 and 1 does not raise an error but I assume it should not give a proper result then (I did not look at the test yet nor the paper).

Reply:
Indeed in the original implementation, they add `1` to the offset by default. So we can make sure that one passes a number >= 1.